src/modules/Login/Login.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import {useFormik} from 'formik';
+import * as Yup from 'yup';
+import {TEMPLATEERRORSTRING} from '@Modules/Registration/Registration';
+import '../Registration/index.scss'
```

Comment:
Weird that you use styles from `Registration` in the `Login` component. Please create own styles related file for `Login`.